.github/workflows/dev-aws-ecr.yml

```diff
@@ -8,54 +8,54 @@ jobs:
+          role-duration-seconds: '3600'
 
-    - name: Login to Amazon ECR
-      id: login-ecr
-      uses: aws-actions/amazon-ecr-login@v1
-    - name: Build, tag, and push image to Amazon ECR
-      id: build-aws-image
-      env:
-        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        ECR_REPOSITORY: portal-client
-        IMAGE_TAG: ${{ github.sha }}
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Build, tag, and push image to Amazon ECR
+        id: build-aws-image
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: portal-client
+          IMAGE_TAG: ${{ github.sha }}
 
-      run: |
-        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
-        docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
-        echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"
-        docker logout $ECR_REGISTRY
+        run: |
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+          echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"
+          docker logout $ECR_REGISTRY
 
-    - name: Download task definition | Update Task Definition | Update ECS Service
-      env:
-        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        ECR_REPOSITORY: portal-client
-        IMAGE_TAG: ${{ github.sha }}
-      run: |
-        ./scripts/deploy-dev-aws.sh
+      - name: Download task definition | Update Task Definition | Update ECS Service
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: portal-client
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          ./scripts/deploy-dev-aws.sh
 
-    - name: Build, tag, and push image to C1 Artifactory
-      id: build-c1-image
-      env:
-        IMAGE_TAG: ${{ github.sha }}
-        C1_REGISTRY: ${{ secrets.C1_REGISTRY }}
-        C1_REPOSITORY: ${{ secrets.C1_REPOSITORY }}
-      run: |
-        sudo sh scripts/add-dod-cas.sh
-        docker login -u portal -p ${{ secrets.C1_ARTIFACTORY_TOKEN }} $C1_REGISTRY
-        docker build -t $C1_REGISTRY/$C1_REPOSITORY:$IMAGE_TAG .
-        docker push $C1_REGISTRY/$C1_REPOSITORY:$IMAGE_TAG
-        echo "::set-output name=image::$C1_REGISTRY/$C1_REPOSITORY:$IMAGE_TAG"
-        docker logout $C1_REGISTRY
+      - name: Build, tag, and push image to C1 Artifactory
+        id: build-c1-image
+        env:
+          IMAGE_TAG: ${{ github.sha }}
+          C1_REGISTRY: ${{ secrets.C1_REGISTRY }}
+          C1_REPOSITORY: ${{ secrets.C1_REPOSITORY }}
+        run: |
+          sudo sh scripts/add-dod-cas.sh
+          docker login -u portal -p ${{ secrets.C1_ARTIFACTORY_TOKEN }} $C1_REGISTRY
+          docker build -t $C1_REGISTRY/$C1_REPOSITORY:$IMAGE_TAG . --build-arg BUILD=$IMAGE_TAG 
```

Comment:
Does the Dockerfile need to be updated to set this argument as an env var?

Reply:
that would be helpful! not sure what happened, looks like all my changes didn't get pushed up. thank you!